package.json

```diff
@@ -38,7 +38,7 @@
       "eslint",
       "prettier --write --plugin-search-dir=."
     ],
-    "*.{css}": [
+    "*.{css,json,html}": [
```

Comment:
I just added this so `prettier` should format any json and html files on commit. Hopefully that reduces the number of prettier errors on the github end of things.

Reply:
looks good!